docs/modules/ROOT/pages/explanations/decisions/mariadb.adoc

```diff
@@ -44,30 +47,24 @@ Some additional Notes:
 * Percona has a MySQL Operator: https://docs.percona.com/percona-operator-for-mysql/pxc/
 * MOCO Operator (MySQL Only): https://github.com/cybozu-go/moco
 ** Has a similar concept to handling clustering like the Patroni framework that is used by Stackgres.
+* New operator, independent from MariaDB project, MIT licensed:  https://github.com/mariadb-operator/mariadb-operator[New Mariadb Operator]
 
```

Comment:
I see no documented reason why the additional operators under "Some additional Notes" haven't been properly analyzed and compared. Please add the reason to the list for future-us to understand why we skipped them.